lib/RoleDetails/RoleDetails.js

```diff
@@ -44,28 +67,70 @@ export const RoleDetails = ({ roleId, path, tenantId }) => {
 
   const { roleDetails: role } = useRoleById(roleId, { tenantId });
   const { mutateAsync: deleteRole } = useDeleteRoleMutation(onClose, sendErrorCallout, { tenantId });
+  const { shareRole } = useInitialRoleSharing(role);
 
-  const [isDeleting, setIsDeleting] = useState(false);
+  const isRoleShared = Boolean(stripes.hasInterface('consortia') && isShared(role?.source));
```

Comment:
for role we should check 'type' field, that should equal to 'CONSORTIUM' value
for policy we should use 'source' that should equal to 'CONSORTIUM'
Better to make this check as case insensitive